src/Blocks/BlockComparator.h

```diff
@@ -170,10 +170,12 @@ class cBlockComparatorHandler final :
 		else if (cBlockSlabHandler::IsAnySlabType(BelowBlock))
 		{
 			// Check if the slab is turned up side down
-			if ((BelowBlockMeta & 0x08) == 0x08)
-			{
-				return true;
-			}
+			return (BelowBlockMeta & 0x08) == 0x08;
+		}
+		else if (IsBlockStair(BelowBlock))
```

Comment:
I think this function should be part of the cBlockStairsHandler, like the IsSlabType above.

Reply:
True, It would be more consistent with the slab function.